agent/pkg/client/credentials.go

```diff
@@ -0,0 +1,244 @@
+/*
+Copyright © 2024 Pulumi Corporation
```

Comment:
Incorrect year

Reply:
True, and I suppose your second pass on linting will fix it.